jest.config.js

```diff
@@ -2,11 +2,13 @@ export default {
   rootDir: 'web_src',
   setupFilesAfterEnv: ['jest-extended/all'],
   testEnvironment: 'jest-environment-jsdom',
-  testMatch: ['<rootDir>/**/*.test.js'],
+  testMatch: ['<rootDir>/**/*.test.js'], // to run tests with ES6 module, node must run with "--experimental-vm-modules"
```

Comment:
We already have this flag in the Makefile, not sure the benefit of putting this here. Maybe at the start of the file, if at all.

Reply:
It's for people who don't use or read the Makefile

Reply:
Then put it at the start of the file.

Reply:
done